hack/packet-provision.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set +e
+
+# Your Packet user account
+if [ "$PACKET_AUTH_TOKEN" == "" ]; then
+    echo "You need to set PACKET_AUTH_TOKEN variable first."
+    echo "Make sure that your SSH key is also set in packet.net"
+    exit 1
+fi
+
+export TF_VAR_environment_id=${ENVIRONMENT_ID:-$(uuidgen | cut -c1-8)}
+
+cd ./prebuild
+case ${1} in
+  "install")
+    ssh_path="$TF_VAR_ssh_key_path"
+    if [ "$TF_VAR_ssh_key_path" == "" ]; then
+        echo -e "\e[33mCreating temporary SSH file\e[0m"
+        ssh-keygen -t rsa -b 4096 -C "temporary packet.net key" -P "" -f "/tmp/packet_id_rsa" -q
+        ssh_path="/tmp/packet_id_rsa.pub"
+    fi
+    terraform init -input=false
+    terraform plan -input=false -out=tfplan.out && terraform apply -input=false -auto-approve tfplan.out
```

Comment:
As we have `-e` this could be two separate commands.

Reply:
Right, I will fix that in a follow-up PR